pkg/apiserver/registry/core/secretbinding/strategy.go

```diff
@@ -65,12 +65,15 @@ func (secretBindingStrategy) ValidateUpdate(_ context.Context, newObj, oldObj ru
 	return validation.ValidateSecretBindingUpdate(newBinding, oldBinding)
 }
 
+const secretBindingDeprecationWarning = "SecretBinding is deprecated and will be disallowed starting with Kubernetes 1.34." +
```

Comment:
```suggestion
const secretBindingDeprecationWarning = "SecretBinding is deprecated and will be disallowed to be used by Shoots using Kubernetes 1.34 or greater." +
```

Reply:
Warnings are expected to be typically below 120 characters 
ref1 - https://kubernetes.io/blog/2020/09/03/warnings/#admission-webhooks
ref2 - https://github.com/kubernetes/kubernetes/blob/28c8381aa5b2a6406dfce3540167d5d9b97b20a7/staging/src/k8s.io/apiserver/pkg/registry/rest/create.go#L68

Reply:
Although the suggestion is within the soft limit I still like the original better (even though technically the other statement is correct). I expect that we remove the secret binding API after 1.33 support is dropped.

Reply:
The original one in this form have the extra information (the kubernetes version) which actually rises more questions. Alternatively, you can use something like:

```suggestion
const secretBindingDeprecationWarning = "SecretBinding is deprecated in favor of CredentialsBinding." +
```

We cannot comply with the limit of 120 characters because of the second part of the warning which is only 158 chars. Maybe we can remove it.